js/skin.js

```diff
@@ -200,16 +200,16 @@ var Skin = createReactClass({
    * @returns {[number, number, number]} array with yaw, roll, pitch
    */
   getDirectionParams: function(_pageX, _pageY) {
-    var pageX = Utils.ensureNumber(_pageX, 0);
-    var pageY = Utils.ensureNumber(_pageY, 0);
-    var dx = pageX - this.state.xVrMouseStart;
-    var dy = pageY - this.state.yVrMouseStart;
-    var maxDegreesX = 90;
-    var maxDegreesY = 120;
-    var degreesForPixelYaw = maxDegreesX / this.state.componentWidth;
-    var degreesForPixelPitch = maxDegreesY / this.state.componentHeight;
-    var yaw = this.getVrViewingDirectionParamValue('yaw') + dx * degreesForPixelYaw;
-    var pitch = this.getVrViewingDirectionParamValue('pitch') + dy * degreesForPixelPitch;
+    let pageX = Utils.ensureNumber(_pageX, 0);
```

Comment:
Not super important, but it looks like none of these are re-assigned, so they could as well be all `const`

Reply:
\+